scripts/bees.coffee

```diff
@@ -0,0 +1,18 @@
+# Description:
```

Comment:
:angry: You had a chance to silently remove this file and you didn't

Reply:
It functions as a mechanism to discourage talking about bees. I find this disappointing.

Reply:
end ralph's bee oppression

Reply:
![bees](http://i.imgur.com/qrLEV.gif)

Reply:
that would bee sweet